apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+                    - $ref: '#/components/schemas/CreateExistedExperiment-200OK'
+                examples:
+                  New_Experiment:
+                    summary: IF_NEW_EXPERIMENT
+                    value: 
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-30T07:17:22.982+00:00  '
+                      params:
+                        resmsgid: 565b07a3a-d59f-47e6-b3d3-76cc5866411c
+                        status: successful
+                      responseCode: OK
+                      result:
+                        stats: {}
+                        request: '#/components/schemas/Experiment-Request/example'
+                        created_on: 1606818148268
+                        last_updated: 1606818148268
+                        status: SUBMITTED
+                        status_msg: Experiment successfully submitted
+                  Existing_Experiment:
+                    summary: If_Experiment_Already_Exists
+                    value: 
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: successful
+                      responseCode: OK
+                      result:
+                       expResponse:
+                         request: '#/components/schemas/Experiment-Request/example'
+                         err: Failed
+                         errorMsg: ExperimentId already exists.                    
+        
+          '400':
+            description: Bad Request for create experiment
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/CreateExperiment-400BadRequest'
+                examples:
+                  EmptyRequest:
+                    summary: Request_Field_Missing
+                    description: Response example when request is missing
+                    value:
+                      $ref: '#/components/schemas/CreateExperiment-400BadRequest/example'
+                  RequiredRequestFieldsEmpty:
+                    summary: Required Fields Missing in Request
+                    description: Response example with all required fields
+                    value:
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: failed
+                        err: CLIENT_ERROR
+                        errorMsg:
+                          status: failed
+                          request.expId: Created By should not be empty
```

Comment:
The field request.expId but the description says Created By should not be empty.